R/object_usage_linter.R

```diff
@@ -98,6 +107,51 @@ object_usage_linter <- function() {
   })
 }
 
+extract_glued_symbols <- function(expr) {
+  glue_calls <- xml2::xml_find_all(
+    expr,
+    xpath = paste0(
+      "descendant::SYMBOL_FUNCTION_CALL[text() = 'glue']/", # a glue() call
```

Comment:
maybe as a TODO: there's also `glue_safe()`, `glue_sql()`, and `glue_col()`, which we could run through the same logic as `glue()`, and `glue_data()` (and similar safe/sql/col variants)

Reply:
`stringr::str_interp()` could also be supported. Adding note.